caps/capabilities.go

```diff
@@ -95,6 +99,16 @@ func NewRepository() *Repository {
 	}
 }
 
+// AddBuiltInTypes adds all built-in types to the repository
+// If any of the additions fail the function panics.
+func (r *Repository) AddBuiltInTypes() {
+	for _, t := range builtInTypes {
+		if err := r.AddType(t); err != nil {
+			panic(err.Error())
```

Comment:
Doesn't feel like this should be a panic. The fact an arbitrary repository has conflicting types is a very plausible scenario. I'd suggest erroring out from LoadBuiltIn. The call site, though, may decide it's a panic situation to get an error when loading built-ins into an empty repository.